docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -756,6 +756,15 @@ If you perform the `SELECT` query between merges, you may get expired data. To a
 - [ttl_only_drop_parts](/docs/en/operations/settings/settings.md/#ttl_only_drop_parts) setting
 
 
+## Disk types
+
+In addition to local block devices, ClickHouse supports other device types through table engines. These are the types:
```

Comment:
This statement is subtly incorrect.
Disk abstraction is different from integration table engines.